core/vm/gas.go

```diff
@@ -51,3 +54,42 @@ func callGas(isEip150 bool, availableGas, base uint64, callCost *big.Int) (uint6
 
 	return callCost.Uint64(), nil
 }
+
+// IntrinsicGas computes the 'intrinsic gas' for a message with the given data.
+func IntrinsicGas(data []byte, contractCreation, homestead, istanbul, isValidatorCreation bool) (uint64, error) {
```

Comment:
just a direct copy, right?

Reply:
A move, yes. It's been moved from `core` to `vm`.